staging/kubeaddons-catalog/templates/ingress.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.ingress.enable }}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "kubeaddons-catalog.fullname" . }}-ingress
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "kubeaddons-catalog.labels" . | indent 4 }}
```

Comment:
Please add the ability to create annotations via values for the ingress controller.